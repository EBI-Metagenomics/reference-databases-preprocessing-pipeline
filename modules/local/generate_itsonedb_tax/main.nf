
process GENERATE_ITSONEDB_TAX {

    label 'light'

    input:
    path fasta
    path uplift
    val version
    val label

    output:
    path("*uplift.final.txt"), emit: uplift_final

    script:
    """
    generate_itsonedb_tax.py -f $fasta -u $uplift -o ./uplift.final.txt
    """

}