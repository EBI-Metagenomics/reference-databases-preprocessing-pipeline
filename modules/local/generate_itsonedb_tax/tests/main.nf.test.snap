{
    "uplift_final": {
        "content": [
            [
                "uplift.final.txt:md5,dfe9111b418e3e0b7b1d60efd1f816cb"
            ]
        ],
        "timestamp": "2023-12-12T15:56:05.042159553"
    }
}